### PR TITLE
virtual/adjust-infrastructure: tweak ebuild for forked rex

### PR DESCRIPTION
**Pull request by xmhd**
tweaked infrastructure ebuild to allow use of either rex or trex, and add some more deps.

### CODE REVIEW

**Comment by monsieurp** [comment]
hey dave please mark this package stable whilst at it :)

**Comment by xmhd** [comment]
> hey dave please mark this package stable whilst at it :)

iirc (it has been a few weeks since I last checked), some of the deps are ~amd64. I will check again when I'm online tomorrow.

File: virtual/adjust-infrastructure/adjust-infrastructure-0.1-r1.ebuild [hunk 1]
```diff
@@ -24,6 +27,8 @@ RDEPEND="
 	dev-perl/Net-CIDR-Lite
 	dev-perl/Number-Bytes-Human
 	dev-perl/Path-Tiny
+	dev-perl/Rex-Interface-Shell-Idrac4
+	dev-perl/Rex-Interface-Shell-Ilo
```
Comment (mcharriere) [inline virtual/adjust-infrastructure/adjust-infrastructure-0.1-r1.ebuild:31]: Could you add the following?
- dev-perl/Rex-Hook-File-Impostor 
- dev-perl/Rex-Hook-File-Diff 
- dev-perl/Template-Toolkit

@monsieurp what do you think?

Comment (monsieurp) [reply virtual/adjust-infrastructure/adjust-infrastructure-0.1-r1.ebuild:31]: 👍

**Review by monsieurp — APPROVED** [review]

**Pull request merged**